content/mtr/mtr.md

```diff
@@ -62,17 +62,17 @@ Commanders are revealed prior to shuffling and presenting decks. A game should n
 
 [Original](https://blogs.magicjudges.org/rules/mtr2-4/)
 
-When time is called, the active player finishes their turn, and there are no additional turns. The game ends when the active player passes their turn. If a non-active player is acting in the end step of the active turn, when time is called, the subsequent turn becomes the final one. A time limit of 20 minutes should be applied to the final turn when time is called.
+When time is called, the active player finishes their turn, and there are no additional turns. The game ends when the active player passes their turn. A time limit of 20 minutes should be applied to the final turn when time is called. If this time limit is reached then the spell or ability that is currently on top of the stack resolves and the rest of the stack is exiled.
```

Comment:
This addition of the top spell or ability resolving probably deserves some discussion, some locations don't have this and just exile the whole stack and others resolve the top item and then exile the stack. These are mostly identical but there is the edge case of the top item of the stack being a Thassa's Oracle trigger or similar "Win the game" effect and this resolving means a player wins that wouldn't have if this had happened under the other version of this rule.